aggregate-snapshot/aggregate-snapshot-domain/src/test/java/uk/gov/justice/domain/snapshot/AggregateSnapshotTest.java

```diff
@@ -26,7 +26,7 @@ public class AggregateSnapshotTest {
     public void shouldCreateAnAggregateSnapshot() throws Exception {
         final TestAggregate aggregate = new TestAggregate("STATE1");
 
-        final AggregateSnapshot<TestAggregate> snapshot = new AggregateSnapshot<>(STREAM_ID, VERSION_ID, aggregate);
+        final AggregateSnapshot<TestAggregate> snapshot = new AggregateSnapshot<>(STREAM_ID, VERSION_ID, aggregate, null);
```

Comment:
I'm not sure I like passing in null like this. I've found nulls usually lead to tears and pain. Can this be an Optional instead?

Reply:
fixed